src/sssom/context.py

```diff
@@ -25,108 +22,79 @@
 )
 
 
-def get_jsonld_context():
-    """Get JSON-LD form of sssom_context variable from auto-generated 'internal_context.py' file.
-
-    [Auto generated from sssom.yaml by jsonldcontextgen.py]
-
-    :return: JSON-LD context
-    """
+def _get_jsonld_context():
+    """Get JSON-LD form of sssom_context variable from auto-generated 'internal_context.py' file."""
     with open(SSSOM_CONTEXT, "r") as c:
         context = json.load(c, strict=False)
-
     return context
 
 
-def get_extended_prefix_map():
-    """Get prefix map from bioregistry (obo.epm.json).
-
-    :return: Prefix map.
-    """
-    converter = Converter.from_extended_prefix_map(EXTENDED_PREFIX_MAP)
-    return {record.prefix: record.uri_prefix for record in converter.records}
+def get_internal_converter() -> Converter:
+    """Get a converter from the SSSOM internal context."""
+    context = _get_jsonld_context()
+    prefix_map = {}
+    for key in context["@context"]:
+        v = context["@context"][key]
+        if isinstance(v, str):
+            prefix_map[key] = v
+        elif isinstance(v, dict):
+            if "@id" in v and "@prefix" in v:
```

Comment:
is this ID syntax actually part of the context map standard? We can upstream this into `curies` if so

Reply:
None of this will be needed anymore when your PR is done.